src/common/redis_module/ray_redis_module.cc

```diff
@@ -717,23 +696,77 @@ int TableAppend_RedisCommand(RedisModuleCtx *ctx,
     // necessary since we implement the log with a sorted set, so all entries
     // must be unique, or else we will have gaps in the log.
     RAY_CHECK(flags == REDISMODULE_ZADD_ADDED) << "Appended a duplicate entry";
-    // Publish a message on the requested pubsub channel if necessary.
-    TablePubsub pubsub_channel = ParseTablePubsub(pubsub_channel_str);
-    if (pubsub_channel != TablePubsub::NO_PUBLISH) {
-      // All other pubsub channels write the data back directly onto the
-      // channel.
-      return PublishTableAdd(ctx, pubsub_channel_str, id, data);
-    } else {
-      return RedisModule_ReplyWithSimpleString(ctx, "OK");
-    }
+
   } else {
     // The requested index did not match the current length of the log. Return
     // an error message as a string.
-    const char *reply = "ERR entry exists";
-    return RedisModule_ReplyWithStringBuffer(ctx, reply, strlen(reply));
+    static const char *reply = "ERR entry exists";
+    RedisModule_ReplyWithStringBuffer(ctx, reply, strlen(reply));
+    return REDISMODULE_ERR;
+  }
+  return REDISMODULE_OK;
```

Comment:
Move this into the `if` block above? Since it's only reachable by that branch.

Reply:
Done, good call